metron-platform/metron-elasticsearch/src/main/config/elasticsearch.properties

```diff
@@ -57,10 +57,13 @@ org.apache.metron.metrics.TelemetryParserBolt.fails=true
 
 bolt.hdfs.batch.size=5000
 bolt.hdfs.field.delimiter=|
+bolt.hdfs.rotation.policy=org.apache.storm.hdfs.bolt.rotation.TimedRotationPolicy
+bolt.hdfs.rotation.policy.units=DAYS
+bolt.hdfs.rotation.policy.count=1
+
 bolt.hdfs.file.rotation.size.in.mb=5
```

Comment:
Those lines are actually added by Ansible in the original path (topologies.yml).  They don't exist with any properties in elasticsearch.properties (which gets bundled up as part of the RPM build).  Because the properties don't exist, the topology will fail.

I just used what Ansible defined as the configs, but they're easy to change if we want to (or if there's some difference between environments I missed).

Otherwise if we want to leave them out of elasticsearch.properties, we'll probably want to do something a little more clever (e.g. make elasticsearch.properties a config file handled through Ambari and use that instead of what the RPM has).  I'm unsure if that file gets used in other manners in other deploys, so I was hesitant to make that change.